app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,7 @@
     <% unless current_page?(root_path) || !(publication || @calendar) || remove_breadcrumbs(content_item) %>
       <%= render 'govuk_publishing_components/components/contextual_breadcrumbs', content_item: content_item_hash %>
     <% end %>
+    <%= render "govuk_web_banners/recruitment_banner" %>
```

Comment:
The commit message about breadcrumbs got me thinking about travel advice pages. They call the same contextual breadcrumb component, so why did breadcrumbs need to be added to their show view for them to show up? They must be being removed by the `remove_breadcrumbs` method. 
Something to fix separately to this change.